tests/acl/null_route/null_route_helper

```diff
@@ -0,0 +1,220 @@
+#!/usr/bin/env python3
+
+"""
+Utility for blocking and unblocking traffic from given source ip_prefix on ACL tables.
+
+The block operation will insert a DENY rule at the top of the table. The unblock operation
+will remove an existing DENY rule that has been created by the block operation (i.e. it does
+NOT insert an ALLOW rule, only removes DENY rules).
+
+This utility depends on L3 ingress ACL table SVI_PROTECT_V4 (for IPv4) and SVI_PROTECT_V6 (for IPv6). 
+Please be sure that those ACL tables are already created.
+
+Example:
+
+Block traffic from 10.2.3.4/32:
+./null_route_helper block 10.2.3.4/32
+
+Unblock all traffic from 10.2.3.4/32:
+./null_route_helper unblock 10.2.3.4/32
+"""
+
+
+from __future__ import print_function
+
+import syslog
+import sys
+import click
+import ipaddress
+
+from swsssdk import ConfigDBConnector
+
+SVI_PROTECT_ACL_TABLES = {
+    4: "SVI_PROTECT_V4",
+    6: "SVI_PROTECT_V6"
+}
+
+CONFIG_DB_ACL_TABLE_TABLE = "ACL_TABLE"
+CONFIG_DB_ACL_RULE_TABLE = "ACL_RULE"
+CONFIG_DB_VLAN_TABLE = "VLAN"
+
+ACTION_ALLOW = "FORWARD"
+ACTION_DENY = "DROP"
+
+# Since SONiC supports multi ACL rules share the same priority, we use 9999 (the highest) for all rules
+EXO_ACL_RULE_PRIORITY = 9999
+# The key of rule will be overridden with EXO_ACL_RULE_ + ip
+EXO_ACL_RULE_PREFIX = 'EXO_BLOCK_RULE_'
```

Comment:
>EXO_BLOCK_RULE_ [](http://example.com/codeflow?start=23&length=15)

This should be for general use case, so the naem should be like RULE_BLOCK_FROM_*